content/200-concepts/100-components/02-prisma-client/053-client-extensions/130-result.mdx

```diff
@@ -68,22 +68,22 @@ const xprisma = prisma.$extends({
   },
 })
 
-const user = await xprisma.user.findFirst()
+const user = await prisma.user.findFirst()
 
 // return the user's full name, such as "John Doe"
 console.log(user.fullName)
 ```
 
 In above example, the input `user` of `compute` is automatically typed according to the object defined in `needs`. `firstName` and `lastName` are of type `string`, because they are specified in `needs`. If they are not specified in `needs`, then they cannot be accessed.
 
-When you call a method in an extension, use the constant name from your `$extends` statement, not `prisma`. In the above example, `xprisma.user.findFirst` computes the virtual field, but `prisma.user.findFirst` would not.
+When you call a method in an extension, use the constant name from your `$extends` statement, not `prisma`. In the above example, `prisma.user.findFirst` computes the virtual field, but `prisma.user.findFirst` would not.
```

Comment:
This comment doesn't make sense anymore since we've renamed `xprisma` to `prisma`

Reply:
If this was the only place where this was highlighted, we now might need to surface this in another place as i is a common pattern how to use extensions.